### ISSUE TITLE
Open File not working

### ISSUE THREAD

**Issue by DumbCO** [JadenGeller/Helium#251]
What am I doing wrong? Just installed version 2.0 on my Mac. When I try to open a file, nothing happens.

**Comment by slashlos** [comment]
Try [Helium 3.0.3596](https://github.com/slashlos/Helium/releases/tag/3.0.3596).

**Comment by DumbCO** [comment]
The link you provided is invalid.

**Comment by DumbCO** [comment]
I've searched everywhere and there appears to be no higher version than the one I have, below 3.0. Thanks

**Comment by slashlos** [comment]
Yes I believe that's true as Jaden hasn't updated in quite a while.  You might try [3.0](https://github.com/slashlos/Helium/releases/download/3.0.3559/Helium.3.0.3599.dmg) which I maintain.

**Comment by DumbCO** [comment]
Excellent. Thank you. Works now. Much appreciated.

**Comment by bashang03** [comment]
sorry to bother but it still doesn't work.
my system version is mojave 10.14.2

> Yes I believe that's true as Jaden hasn't updated in quite a while. You might try [3.0](https://github.com/slashlos/Helium/releases/download/3.0.3559/Helium.3.0.3599.dmg) which I maintain.

**Comment by orvtnj** [comment]
@slashlos could you please upload the 3.0 one more time? The link doesn't work anymore. Thank you so much

**Comment by slashlos** [comment]
[Helium 3.0.4125](https://github.com/slashlos/Helium/releases/tag/4125) still resolves, the .dmg is ~25MB.  

Just verified on a new mini running 10.12 - the lowest version I support.  Enjoy.
